### DISCUSSION TITLE
Deploy arm for wingtip

### DISCUSSION THREAD

**Discussion by hubbabubban74** [Azure/Enterprise-Scale#358] [General] [answered]
Hi,
I´m a newbie to arm and azure and when I try to deploy CAF enterprise-scale landing zone I get the following error.

New-AzDeployment : 18:50:43 - Error: Code=InvalidTemplate; Message=Deployment template validation failed: 'The template variable 'deploymentUris' is not valid: The language expression property 'templateLink' doesn't exist, available properties
 are 'template, templateHash, parameters, mode, provisioningState'.. Please see https://aka.ms/arm-template-expressions for usage details.'.

Is there any tutorial how to deploy this?

**Comment by krnese** [comment, marked as answer]
How are you trying to deploy? The 'Deploy to Azure' button will take you through the portal experience to deploy the reference implementation into your Azure tenant. To setup the required permission prior to deploying, see the instructions @ https://github.com/Azure/Enterprise-Scale/blob/main/docs/EnterpriseScale-Setup-azure.md
